### PR TITLE
Add the memory swappiness tuning support to libcontainer

### PR DESCRIPTION
**Pull request by ktraghavendra**
Signed-off-by: Raghavendra K T <raghavendra.kt@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by dqminh** [comment]
LGTM

File: configs/cgroup.go [hunk 1]
```diff
@@ -87,6 +87,9 @@ type Cgroup struct {
 	// Whether to disable OOM Killer
 	OomKillDisable bool `json:"oom_kill_disable"`
 
+	// Tuning swappiness behaviour per cgroup
+	MemSwappiness int64 `json:"swappiness"`
```
Comment (LK4D4) [inline configs/cgroup.go]: Can we call it `MemorySwappiness` for consistency? and move it to another `Memory..` fields?
Also as I see we don't use short names in json, so should be `memory_swappiness`

**Comment by LK4D4** [comment]
I have naming nits, otherwise patch is great.

File: configs/cgroup.go (hunk 1 above)
Comment (ktraghavendra) [inline configs/cgroup.go]: Thanks @dqminh @LK4D4 for review. Changing the naming and pushing to branch

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**